base/exports.jl

```diff
@@ -117,13 +117,15 @@ export
     Cwstring,
 
 # Exceptions
-    DimensionMismatch,
     CapturedException,
     CompositeException,
+    DimensionMismatch,
     EOFError,
     InvalidStateException,
     KeyError,
     MissingException,
+    ParseError,
+    ProcessExitedError,
```

Comment:
Most (but not all) other exception types have "exception" in the name instead of "error." In keeping with that, I'd call this `ProcessExitedException`, which I think sounds better anyway.

Reply:
I agree.
Exceptions have a mix of naming either `FooException`, or `BarError`.
I feel like we should have a convention that they are different (e.g. `Errors` are probably programmers fault. Where as who knows what causes `Exceptions`s. Or perhaps that `rrors` are normally n ot recoverable. idk).

But yet, lets name this an exception.